app/views/items/_item.html.erb

```diff
@@ -0,0 +1,5 @@
+    <li class=" list-group-item"><%= item.description %></li>
+
+    <% @user.items do |item| %>
```

Comment:
Don't forget that we are already looping through all the items in users/show, so you don't need to loop through them here. The link can just go right in the li.